tests/test_integrity.py

```diff
@@ -490,6 +490,15 @@ def test_spectra(db):
     assert len(flux_unit_fail) == 0, f'Some flux units did not resolve: {flux_unit_fail}'
 
 
+def test_special_characters(db):
+    # This test asserts that no special unicode characters are in the database
+    # This can be expanded with additional characters we want to avoid
+    bad_characters = ['\u2013']
```

Comment:
Lets also add ó since we you just got rid of it! And another common on é
\u00f3 ó
\u00e9 é
```suggestion
    bad_characters = ['\u2013', '\u00f3', '\u00e9']
```

Reply:
It looks a little more complex now, since I wanted to avoid failing on descriptions and some paper titles do have accents, but this is working now. Includes a,e,i,o,u